JMMServer/JMMServiceImplementationKodi.cs

```diff
@@ -639,6 +640,9 @@ private System.IO.Stream GetGroupsFromFilter(int userid, string GroupFilterId)
                                         }
                                         v.OriginalTitle = v.OriginalTitle.Substring(0, v.OriginalTitle.Length - 1);
                                         //proper naming end
+                                        List<AnimeSeries> sers = grp.GetAllSeries();
+                                        AnimeSeries ser = sers[0];
+                                        v.totalLocal = ser.GetAnimeEpisodesCountWithVideoLocal();
```

Comment:
I fixed it, there was an error in getAnimeEpisodesCountWithVideoLocal. Unfortunately, the system is flawed to begin with, as it returns the entire series total, not just for each level.
```
-Group Filters (no info)
---Group (first underlying series info, not total)
-----Series (total for whole series, this will be the only correct value)
-------Credits (This is counted in the watched state, causing everything else to be wrong)
-------Episodes (This will show an incorrect value, as it takes total for the series including specials and credits)
-------Specials (This will be incorrect, as it takes the total including episodes and credits)
```